reference/informative/informB15_connection4.rst

```diff
@@ -10,7 +10,7 @@
 
     You may only have one :code:`connection` any two :code:`components`, regardless of which is specified as :code:`component_1` and which is specified as :code:`component_2`.
     If you have found duplicate :code:`connection` elements, simply merge their contents - a :code:`connection` can contain any number of :code:`map_variables` children.
-    Make sure that the order in which you spe **TODO**
+    Make sure that the order in which you specify the :code:`component` attributes matches the order in which you specify their child :code:`variable` items too: ie, all :code:`variable_1` items must be within the :code:`component_1` component and vice versa.
```

Comment:
"i.e." not "ie".

Reply:
Yep, the informative stuff needs a good going over, it's nowhere near done.  All the PRs here are concentrating on the normative spec instead.  I'll do this kind of thing once, later :)

Reply:
Ok, fair enough.